### DISCUSSION TITLE
Chart tool tip with value unit

### DISCUSSION THREAD

**Discussion by MoritzSalein** [ludeknovy/jtl-reporter#148] [Ideas]
Every chart tool tip should be with the value unit as the detailed request chart.
![2022-02-25 10_12_06-JtlReporter](https://user-images.githubusercontent.com/83296020/155689822-346838ad-79ac-4e81-9e47-9368e3a7fbd7.png)
![2022-02-25 10_22_34-JtlReporter](https://user-images.githubusercontent.com/83296020/155689950-0eb6b560-437f-4876-9076-69542240c312.png)

**Comment by ludeknovy** [comment]
https://github.com/ludeknovy/jtl-reporter/issues/152
